docs/config.md

```diff
@@ -140,6 +140,7 @@ due to lack of stable semantic convention.
 | `STACKEXCHANGEREDIS`  | [StackExchange.Redis](https://www.nuget.org/packages/StackExchange.Redis) **Not supported on .NET Framework**                                                                                   | ≥2.0.405 < 3.0.0   | source & bytecode       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
 | `WCF`                 | [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **No support for server side on .NET**. For configuration see [WCF Instrumentation Configuration](wcf-config.md)      | * \[4\]            | source                  | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
 | `WCFCLIENT`           | [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **Supported on .NET Framework**.                                                                                      | ≥4.0.0.0 < 5.0.0.0 | source & bytecode       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `WCFSERVICE`          | [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **Supported on .NET Framework**.                                                                                      | ≥4.0.0.0 < 5.0.0.0 | source & bytecode       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
Can we change both remark to `Not supported on .NET`?

```suggestion
| `WCFCLIENT`           | [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **Not supported on .NET**.                                                                                            | ≥4.0.0.0 < 5.0.0.0 | source & bytecode       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
| `WCFSERVICE`          | [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **Not supported on .NET**.                                                                                            | ≥4.0.0.0 < 5.0.0.0 | source & bytecode       | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Just for consistency with other entries.